scripts/common/install-latest-librdkafka.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+#### Install latest librdkafka, right from the GIT repo.
+
+### Define the version of kafka we are pinning to.
+kafka_version=1.5.0
+
+
+### Install librdkafka
+## Install dependencies
+apk add --no-cache libressl-dev || exit 2
+apk add --no-cache cyrus-sasl-dev || exit 2
+apk add --no-cache zstd-dev || exit 2
+apk add --no-cache zlib-dev || exit 2
+apk add --no-cache git || exit 2
+apk add --no-cache make || exit 2
+apk add --no-cache gcc || exit 2
+apk add --no-cache g++ || exit 2
+
+## Fetch librdkafka from github
+git clone https://github.com/edenhill/librdkafka.git || exit 2
```

Comment:
There's a tarball for these that would save needing to do a full git clone. See https://github.com/edenhill/librdkafka/archive/v1.5.0.tar.gz.

Reply:
(came here to say this, use a release 😂 )

Reply:
I was following the official [instructions](https://github.com/edenhill/librdkafka#build-from-source) on the website, which do not mention using a release. Having said that, I can use a tarball.